extensions/blocks/podcast-player/edit.js

```diff
@@ -203,20 +219,51 @@ const PodcastPlayerEdit = ( {
 						onChange={ value => setAttributes( { showEpisodeDescription: value } ) }
 					/>
 				</PanelBody>
+				<PanelColorSettings
+					title={ __( 'Color Settings', 'jetpack' ) }
+					colorSettings={ [
+						{
+							value: primaryColorProp.color,
+							onChange: setPrimaryColor,
+							label: __( 'Primary Color', 'jetpack' ),
+						},
+						{
+							value: secondaryColorProp.color,
+							onChange: setSecondaryColor,
+							label: __( 'Secondary Color', 'jetpack' ),
+						},
+						{
+							value: backgroundColorProp.color,
+							onChange: setBackgroundColor,
+							label: __( 'Background Color', 'jetpack' ),
+						},
+					] }
+				>
+					<ContrastChecker
+						isLargeText={ false }
+						textColor={ secondaryColorProp.color }
+						backgroundColor={ backgroundColorProp.color }
+						fallbackBackgroundColor
+						fallbackTextColor
+					/>
+				</PanelColorSettings>
 			</InspectorControls>
 			<div className={ className }>
 				<PodcastPlayer
+					attributes={ validatedAttributes }
 					tracks={ feedData.tracks }
 					cover={ feedData.cover }
 					title={ feedData.title }
 					link={ feedData.link }
-					itemsToShow={ itemsToShow }
-					showEpisodeDescription={ showEpisodeDescription }
-					showCoverArt={ showCoverArt }
+					coverArt={ feedData.coverArt }
```

Comment:
Should it be `showCoverArt` here, to match uses elsewhere?

Reply:
If I'm not wrong `coverArt` is the source of the cover art, while `showCoverArt` is the block attribute which defines if the cover art should be shown or not.

Reply:
We use plain `cover` in API and all components for a while now to hold the image cover URL. You can see it a few lines above:

`cover={ feedData.cover }`

this `coverArt` line should be removed.